fontir/src/serde.rs

```diff
@@ -63,6 +63,48 @@ impl From<StaticMetadata> for StaticMetadataSerdeRepr {
     }
 }
 
+// The HashMap<NormalizedLocation, GlyphInstance> seems to throw serde for a loop sometimes
```

Comment:
I think this specific change deserved its own issue/PR instead of being placed inside this large and seemingly unrelated commit..